tiny-java-containers/setup-musl.sh

```diff
@@ -1,22 +1,34 @@
-#!/bin/sh
-
+#!/usr/bin/env bash
 set -e
 
-ZLIB_VERSION=1.2.13
-TOOLCHAIN_DIR=`pwd`/x86_64-linux-musl-native
+# Specify an installation directory for musl:
+export MUSL_HOME=$PWD/musl-toolchain
+
+# Download musl and zlib sources:
+curl -O https://musl.libc.org/releases/musl-1.2.4.tar.gz
+curl -O https://zlib.net/fossils/zlib-1.2.13.tar.gz
+
+# Build musl from source
+tar -xzf musl-1.2.4.tar.gz
+rm musl-1.2.4.tar.gz
+pushd musl-1.2.4
+./configure --prefix=$MUSL_HOME --static
+make 
+make install
+popd
+
+# Install a symlink for use by native-image
+ln -s $MUSL_HOME/bin/musl-gcc $MUSL_HOME/bin/x86_64-linux-musl-gcc
```

Comment:
The native-image tool doesn't find `x86_64-linux-musl-gcc` via the symlink. @fniephaus, @shaunsmith, any idea how to tweak things around?
 ```
 [1/8] Initializing...                                                                                    (0.0s @ 0.10GB)
Error: Default native-compiler executable 'x86_64-linux-musl-gcc' not found via environment variable PATH
Error: To prevent native-toolchain checking provide command-line option -H:-CheckToolchain
```